inc/taxonomies.php

```diff
@@ -62,16 +62,6 @@ function largo_custom_taxonomies() {
 
 	$termsDefinitions = array(
 		array(
-			'name' => __( 'Sidebar Featured Widget', 'largo' ),
```

Comment:
If these terms aren't registered, then this function will create the terms down in the `foreach ( $largoProminenceTerms as $term ) {` section with a `wp_insert_term()`, and after that point they exist in the database.

If they are registered, then this function does nothing.

So, yes, they will continue to exist.